main/tasks/stratum_task.c

```diff
@@ -344,6 +416,15 @@ void stratum_task(void * pvParameters)
             } else if (stratum_api_v1_message.method == CLIENT_RECONNECT) {
                 ESP_LOGE(TAG, "Pool requested client reconnect...");
                 stratum_close_connection(GLOBAL_STATE);
+                if (!GLOBAL_STATE->mining_enabled) {
```

Comment:
Why is this check and the stop block so many times in `stratum_task.c`? Is it possible to have one stop block per task? I see it logs a different stop reason, but in all cases an error is already logged, so having a specific stop reason doesn't add a lot.